lib/measures/urban_geometry_creation/measure.rb

```diff
@@ -188,9 +201,20 @@ def run(model, runner, user_arguments)
       @runner.registerError("Unknown feature type '#{feature.type}'")
       return false
     end
-
+#
     # transfer data from previous model
-    stories = URBANopt::GeoJSON::Model.transfer_prev_model_data(model, space_types)
```

Comment:
@tanushree04 did you look at `transfer_prev_model_data`?  Seems like it might have been doing the right thing?

https://github.com/urbanopt/urbanopt-geojson-gem/blob/c3302d17ab3b53bb535fc10c0db3d10fd52b4174/lib/urbanopt/geojson/model.rb#L84

Reply:
Calling from `transfer_prev_model_data` worked. I can commit the changes.